api/base/filters.py

```diff
@@ -44,10 +78,6 @@ class ODMFilterMixin(object):
 
     # TODO Handle simple and complex non-standard fields
 
-    TRUTHY = set(['true', 'True', 1, '1'])
-    FALSY = set(['false', 'False', 0, '0'])
-    DEFAULT_OPERATOR = 'eq'
-
     # For the field_comparison_operators, instances can be a class or a tuple of classes
     field_comparison_operators = [
```

Comment:
A dict seems to be more appropriate for `field_comparison_operators` than a `list`.